dolphinscheduler-ui/src/js/conf/home/pages/dag/_source/formModel/formModel.vue

```diff
@@ -550,7 +554,18 @@
         backfillRefresh: true,
         // whether this is a new Task
         isNewCreate: true,
-        tasksTypeList: Object.keys(tasksType)
+        tasksTypeList: Object.keys(tasksType),
+        helpUrlEnable: function (typeKey) {
+          let type = tasksType[typeKey]
+          if (type) {
+            let disabled = !!type.helperLinkDisable
+            return !disabled
+          }
+          return false
+        },
+        helpUrl: function (tskType) {
```

Comment:
tasktype => {}

Here `tsktype` should be replaced with `tasktype`.

Reply:
ok